src/install.rs

```diff
@@ -40,6 +40,14 @@ pub fn install(config: &InstallConfig) -> Result<()> {
         }
     }
 
+    // if an Ignition config is provided, check upfront that we can open it
```

Comment:
optional: maybe augment the comment with something like: "so we don't waste time downloading the disk image first"

Reply:
Augmented!